app/javascript/app/data/constants.js

```diff
@@ -310,6 +310,13 @@ export const DATA_EXPLORER_SECTION_NAMES = {
   'ndc-content': 'ndc_content'
 };
 
+export const DATA_EXPLORER_SECTION_LABELS = {
```

Comment:
Maybe we can rename DATA_EXPLORER_SECTION_NAMES to URIS or maybe even merge them as they have the same keys. Having Labels and Names together could be confusing

Reply:
If you agree I'll go with the change of the constant to `DATA_EXPLORER_SECTION_URIS`.

Reply:
⚡️ done